cli/src/main/java/com/liferay/blade/cli/command/WatchArgs.java

```diff
@@ -16,14 +16,85 @@
 
 package com.liferay.blade.cli.command;
 
+import com.beust.jcommander.Parameter;
 import com.beust.jcommander.Parameters;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 /**
- * @author Gregory Amerson
+ * @author David Truong
  */
 @Parameters(
 	commandDescription = "Watch for changes to workspace projects and automatically build, deploy, or update them into running Liferay instance.",
 	commandNames = "watch"
 )
 public class WatchArgs extends BaseArgs {
+
+	public List<String> getFastPaths() {
+		return Stream.concat(
+			_defaultFastPaths.stream(), _fastPaths.stream()
+		).collect(
+			Collectors.toList()
+		);
+	}
+
+	public List<String> getIgnorePaths() {
+		return Stream.concat(
+			_defaultIgnorePaths.stream(), _ignorePaths.stream()
+		).collect(
+			Collectors.toList()
+		);
+	}
+
+	public List<String> getProjectPaths() {
+		return Stream.concat(
+			_defaultProjectPaths.stream(), _projectPaths.stream()
+		).collect(
+			Collectors.toList()
+		);
+	}
+
+	public boolean isSkipInit() {
+		return _skipInit;
+	}
+
+	public void setFastPaths(List<String> fastPaths) {
+		_fastPaths.addAll(fastPaths);
+	}
+
+	public void setIgnorePaths(List<String> ignorePaths) {
+		_ignorePaths.addAll(ignorePaths);
+	}
+
+	public void setProjectPaths(List<String> projectPaths) {
+		_projectPaths.addAll(projectPaths);
+	}
+
+	public void setSkipInit(boolean skipInit) {
+		_skipInit = skipInit;
+	}
+
+	private static final List<String> _defaultFastPaths = Arrays.asList(
+		"**/*.css", "**/*.js", "**/*.jsp", "**/*.map", "**/*.scss");
+	private static final List<String> _defaultIgnorePaths = Arrays.asList(
+		".gradle", ".idea", ".settings", "**/.sass-cache", "**/build", "**/classes", "**/dist", "**/liferay-theme.json",
+		"**/node_modules", "**/liferay-npm-bundler-report.html", "**/target", "bundles", "gradle");
+	private static final List<String> _defaultProjectPaths = Arrays.asList("src");
+
+	@Parameter(description = "File paths that will use deployFast instead of deploy.", names = {"-f", "--fast-paths"})
+	private List<String> _fastPaths = Collections.emptyList();
```

Comment:
`= new ArrayList<>();` as `Collections.emptyList()` is immutable.

Reply:
Didn't know that!